x-pack/filebeat/input/entityanalytics/provider/azuread/azure.go

```diff
@@ -165,6 +165,21 @@ func (p *azure) runFullSync(inputCtx v2.Context, store *kvstore.Store, client be
 		tracker.Wait()
 	}
 
+	if len(state.devices) != 0 {
+		tracker := kvstore.NewTxTracker(ctx)
+
```

Comment:
I'm wondering, do we want a single set of writer markers to bound both users and devices? If we want them separate, do we want to attach any sort of field to indicate what the write markers are associated with? (device or user).

Thoughts, @andrewkroh?

Reply:
Good point, either is acceptable to me. If it is two separate marks then someone who is querying would need to be know if it was associated to the users or devices.

Reply:
ISTM a single set makes more sense. devices are users are really pretty much the same kind of entity (at the moment more so, with the TODO additions less so).